packages/palette/src/elements/Select/Select.tsx

```diff
@@ -195,12 +195,17 @@ const SmallSelectContainer = styled.div<SelectProps>`
 const SelectSmallContainer = styled.div<SelectProps>`
   position: relative;
 
+  label {
+    padding: 0;
+    margin: 0;
+  }
+
   select {
     ${hideDefaultSkin};
     background-color: ${color("black10")};
     border-radius: 2px;
+    ${themeGet("fontFamily.sans.medium")};
```

Comment:
Strangely this `themeGet` here seems to include `font-family: ` and not just the value for font. Not sure why?

Reply:
Can you elaborate a bit?

Reply:
I was expecting to write something like `font-family: ${themeGet("fontFamily.sans.medium")};` but that did not really work and inspecting stuff I noticed that it did not work because the output html looked like `font-family: font-family:Unica77LLWebMedium,'Helvetica Neue',Helvetica,Arial,sans-serif;` so I removed the original `font-family:` and things look right now. It is confusing to me though.

Reply:
That's just a way to quickly get at the main Theme.tsx object, so not too sure.